extensions/amp-list/0.1/amp-list.js

```diff
@@ -461,10 +458,13 @@ export class AmpList extends AMP.BaseElement {
     }
 
     return fetch.catch(error => {
+      const actions = Services.actionServiceForDoc(this.element);
+      actions.trigger(this.element, 'fetch-error', null, ActionTrust.LOW);
```

Comment:
okay that this may fire multiple times in case of infinite scroll? 
- If not, it can just be moved below `if (opt_append)`
- If yes, we may wanna have an event object with more info (e.g. which load run it failed)

Reply:
Yea, I think it makes more sense that way. 

> If yes, we may wanna have an event object with more info

Perhaps, let's wait for that FR. :)